model/schema_types.go

```diff
@@ -74,8 +74,10 @@ const (
 	MultiBandImageType = "multiband_image"
 	// TimeSeriesType is the schema type for timeseries values
 	TimeSeriesType = "timeseries"
-	// GeoCoordinateType is the schema type for timeseries values
+	// GeoCoordinateType is the schema type for geocoordinate values
 	GeoCoordinateType = "geocoordinate"
+	// RemoteSensingType is the schema type for remote sensing values
+	RemoteSensingType = "remote_sensing"
```

Comment:
Did we settle on a need for remote sensing and multiband images to be 2 separate types?

Reply:
I think for now we should keep them separate and see if different dataset types still benefit from the remote sensing grouping (could be RGB rather than 12 band for example).